### ISSUE TITLE
Docs with version switcher

### ISSUE THREAD

**Issue by kmuehlbauer** [wradlib/wradlib#23]
Hosting the docs on github is fine, but hosting multiple versions seems a bit difficult. We could host on rtd and lets do the homework there, but this doesn't work for wradlibs plotting images to the docs.

tl;rd: An easy way to implement version switching in the docs is presented.

Here is what I suggest:

- we need a preliminar docs setup (like this one here, which I prepared http://kmuehlbauer.github.io/wradlib-docs-testing/, try hovering over the version number in the left navbar) 
- we need a 'wradlib.css' in wradlib/doc/source/_static which adds the css drop down to the rtd 'style.css'
- we need a 'layout.html' in wradlib/doc/source/_templates/ which overrides some rtd logic and adds the html drop down 

The layout.html for the current setup looks like this (excerpt).
```
 <div class="version-content">
      <a href="../devel/">devel</a>
      <a href="../0.7.0/">0.7.0</a>
      <a href="../0.6.0/">0.6.0</a>
 </div>
```

The workflow would be for 

- devel-releases: - clone the docs-repo, create the docs, overwrite devel-folder, commit add, push 
- tag-releases: -add tag-href in layout.html, -clone the docs-repo, create the docs, add tag-folder, commit add, push

With this setup the docs index.html points to the devel, also the css drop down (via layout.html). So the devel-version can be reached from every existing doc-version via the drop down.

Two inconveniences: 
- The css-drop-down cannot be extended for older versions. This would mean using javascript and the like. But you can get to devel-version at any time.
- We have to add the additional tag-line in layout.html in the sources. So this has to be made before the version-tag is added to the repo. If this is forgotten for some reason, the version will be missing in the css drop-down. But this could be healed.

Ideas? Suggestions?

**Comment by kmuehlbauer** [comment]
I have added the necessary changes with #22 . The workflow is a bit simpler now.

The `index.html` is pointing to `latest/` subfolder in any case. 
Every created doc gets a drop down pointing to the known (at creation time) tagged versions and 'latest'. If a tagged doc is created, a symbolic link points 'latest' to the tagged doc.

I already set up the [wradlib-docs](http://wradlib.org/wradlib-docs/index.html).
